js/views/notificationVw.js

```diff
@@ -0,0 +1,56 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    moment = require('moment'),
+    loadTemplate = require('../utils/loadTemplate');
+
+module.exports = Backbone.View.extend({
+
+  className: "notification flexRow",
+
+  events: {
+    'click .js-avatar': 'avatarClick',
+    'click .js-username': 'avatarClick'
+  },
+
+  initialize: function(){
+    var self=this;
```

Comment:
You don't need self=this if it isn't used in the object. 

It's just needed in situations where you have a callback function that has it's own scope, so it's this is no longer the object's this.